### PR TITLE
Add website v0.1

### PR DESCRIPTION
**Pull request by tuhaihe**

### CODE REVIEW

**Comment by tuhaihe** [comment]
@TomShawn @fengyuchaochao PTAL, thanks!

File: blog/2023-05-19-welcome/index.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+slug: welcome
+title: Welcome
+authors: [cloudberrydbteam]
+tags: [database, cloudberrydb]
+---
+
+Greetings and Welcome to the Cloudberry Database Blog! We are pleased to inform you that our ongoing efforts with regards to the Cloudberry Database open source project have been steadily progressing.
```
Comment (TomShawn) [inline blog/2023-05-19-welcome/index.md:8]: Comment from Grammarly.
```suggestion
Greetings and welcome to the Cloudberry Database Blog! We are pleased to inform you that our ongoing efforts with regard to the Cloudberry Database open-source project have been steadily progressing.
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Comment by tuhaihe** [comment]
Let's ship it!

**Pull request merged**